.travis.yml

```diff
@@ -1,8 +1,10 @@
 language: go
 go:
   - tip
-  - 1.6.2
-  - 1.5.4
+  - 1.8
+  - 1.7
+  - 1.6
+  - 1.5
```

Comment:
If you add `.x` to the end of these, Travis will resolve that to the latest minor release of each version (ie, `1.8.x` will be just `1.8`, but `1.7.x` will be `1.7.5`)